tests/app/main/views/test_email_branding.py

```diff
@@ -174,10 +174,7 @@ def test_cant_create_new_email_branding_with_unknown_domain(
         'southend.essex.gov.uk',
         'Not an organisation-level domain (use essex.gov.uk if appropriate)',
     ),
-    pytest.mark.xfail(
-        ('voa.gov.uk', ''),
-        raises=AssertionError
-    ),
+    pytest.param('voa.gov.uk', '', marks=pytest.mark.xfail(raises=AssertionError)),
```

Comment:
Not a dealbreaker, but I’d prefer to keep this on multiple lines so it can be scanned alongside the preceding examples:
```suggestion
    pytest.param(
        'voa.gov.uk',
        '',
        marks=pytest.mark.xfail(raises=AssertionError)
    ),
```